src/ws/cockpit.socket.in

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Cockpit Web Service Socket
+Documentation=man:cockpit-ws(8)
+
+[Socket]
+ListenStream=9090
+ExecStartPost=-/bin/ln -sf @datadir@/@PACKAGE@/issue/active.issue @datadir@/@PACKAGE@/issue/cockpit.issue
+ExecStopPost=-/bin/ln -sf @datadir@/@PACKAGE@/issue/inactive.issue @datadir@/@PACKAGE@/issue/cockpit.issue
```

Comment:
See above, must not change stuff in `/usr` at runtime. I suggest to go via `/run` or just directly set up the /etc/issue.d symlink.